src/OpenTelemetry.Exporter.Prometheus.HttpListener/PrometheusHttpListener.cs

```diff
@@ -18,37 +18,37 @@
 using System.Net;
 using System.Threading;
 using System.Threading.Tasks;
+using OpenTelemetry.Exporter.Prometheus.Shared;
 using OpenTelemetry.Internal;
 
-namespace OpenTelemetry.Exporter.Prometheus
+namespace OpenTelemetry.Exporter.Prometheus.HttpListener
 {
-    /// <summary>
-    /// An HTTP listener used to expose Prometheus metrics.
-    /// </summary>
-    internal sealed class PrometheusExporterHttpServer : IDisposable
+    public sealed class PrometheusHttpListener : IDisposable
```

Comment:
Does this need to be public?

Reply:
Not really. Extension methods should be the way to go. Fixed.